templates/superset_config.py

```diff
@@ -101,6 +101,11 @@ class CeleryConfig(object):
 TIME_ROTATE_LOG_LEVEL = "DEBUG"
 ENABLE_TIME_ROTATE = True
 
+# html santization
+HTML_SANITIZATION = os.getenv("HTML_SANITIZATION")
+if HTML_SANITIZATION and os.getenv("HTML_SANITIZATION_SCHEMA_EXTENSIONS"):
```

Comment:
If `HTML_SANITIZATION` is required for the other, get both variables in with `getenv` and only use the latter if the former is set, wherever it is used. You would avoid the double call to `getenv` which is a veeeeery big problem (it is ugly).

Reply:
You are right, I have made it less ugly for you!